### PR TITLE
Add a README.md for CDAP Support Bundle script

### PR DESCRIPTION
**Pull request by shourya116**

### CODE REVIEW

File: cdap-support-bundle/bin/collect_support_bundle.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+-------------
+
+This script can be used to troubleshoot issues with CDAP system services or to get information about CDAP installation. 
+
+**Command Line Arguments**
+--------------------------
+
+The CDAP Support Bundle Script takes the following command-line arguments:
+
+1. kube-namespace(n): The Kubernetes namespace where CDAP is installed.
```
Comment (rmstar) [inline cdap-support-bundle/bin/collect_support_bundle.md]: let's explicitly specify the arg names:
e.g: --kube-namepace, -n : the Kubernetes namespace where CDAP is installed

Comment (rmstar) [reply cdap-support-bundle/bin/collect_support_bundle.md]: Also specify the default values, if any, for each of these params

File: cdap-support-bundle/bin/collect_support_bundle.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+**Support Bundle Script Output**
+--------------------------------
+
+The Support Bundle Script iterates through a list of commands and performs different actions depending on the conditions and settings in each command.
+
+At the end, a tarball file will be created. This file will contain the logs for _**config_file**_, _**pod_info**_, _**job_info**_, _**config_map_info**_, _**deployment_info**_, _**node_info**_, _**stateful_set_info**_, _**services_info**_, _**memory_info**_, _**cdap_services_log**_, _**cdap_services_status**_, and _**runtime_namespace_logs**_.
+
+If you want to get the logs for pipeline runs that failed, use the parameters **pipeline-name (-l)**, **run-id (-r)**, and **runtime-namespace (-t)**. 
+
+Below is the command for adding the logs for the pipeline runs:
```
Comment (rmstar) [inline cdap-support-bundle/bin/collect_support_bundle.md]: change: adding -> collecting

File: cdap-support-bundle/bin/collect_support_bundle.md (hunk 1 above)
Comment (shourya116) [reply cdap-support-bundle/bin/collect_support_bundle.md]: Done.

File: cdap-support-bundle/bin/collect_support_bundle.md (hunk 2 above)
Comment (shourya116) [reply cdap-support-bundle/bin/collect_support_bundle.md]: Done.

**Review by rmstar — APPROVED** [review]

**Pull request merged**